src/structures/BaseCommandInteraction.js

```diff
@@ -182,6 +182,8 @@ class BaseCommandInteraction extends Interaction {
   editReply() {}
   deleteReply() {}
   followUp() {}
+  showModal() {}
+  awaitModalSubmit() {}
```

Comment:
this wasn't added to the v14 PR 👀

Reply:
I know, Siris said he planned on it being a separate PR I think.